daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/types/Ledger.scala

```diff
@@ -826,6 +831,8 @@ object Ledger {
       tr: Transaction.Transaction,
   ): EnrichedTransaction = {
 
+    val localContracts = tr.localContracts
```

Comment:
It was probably fun to keep `tr.localContracts` but ok.

Reply:
the `localContracts` method used a `def`. Should I change it to `lazy val` for instance

Reply:
`localContracts` is a method. Do you think I should make it a (lazy) value ?